golem-rib/src/type_registry.rs

```diff
@@ -226,7 +229,44 @@ mod internal {
                 }
             }
 
-            _ => {}
```

Comment:
As I do further tickets, I will ensure this `_` is removed. Even if it results in some duplicate looking code, it ensures correctness and accidentally missing out on required cases.